src/utils/hooks/api/use-get-prices.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+import { useQuery } from 'react-query';
+import { Prices } from 'common/types/util.types';
+import { PRICES_URL } from '../../../constants';
+
+const useGetPrices = (): Prices | undefined => {
```

Comment:
Im wonder if in the future we will always want to get prices on initial render. An alternative would be returning data with a function that would execute the fetch. Something like: 

```
return {
data: ...,
loading:...,
error:...,
fetch:...
}
```

Reply:
Do not take this into consideration for this PR btw :)